pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/answer/ActivityInstanceSelectAnswer.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.ddp.model.activity.instance.answer;
+
+import com.google.gson.annotations.SerializedName;
+import org.broadinstitute.ddp.model.activity.types.QuestionType;
+import org.jdbi.v3.core.mapper.reflect.JdbiConstructor;
+
+import javax.validation.constraints.NotNull;
+
+public class ActivityInstanceSelectAnswer extends Answer<String> {
+
+    @NotNull
+    @SerializedName("value")
+    protected String value;
+
+    @JdbiConstructor
+    public ActivityInstanceSelectAnswer(Long answerId, String questionStableId, String answerGuid, String value) {
+        super(QuestionType.ACTIVITY_INSTANCE_SELECT, answerId, questionStableId, answerGuid);
+        this.value = value;
+    }
+
+    public ActivityInstanceSelectAnswer(Long answerId, String questionStableId, String answerGuid, String value, String actInstanceGuid) {
+        super(QuestionType.ACTIVITY_INSTANCE_SELECT, answerId, questionStableId, answerGuid, actInstanceGuid);
+        this.value = value;
+    }
+
+    @Override
+    public String getValue() {
+        return value;
+    }
+
+    @Override
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return value == null || value.isEmpty();
```

Comment:
Yep, but we have to implement this method. See other classes implementing `Answerable` interface

Reply:
I'm not certain we're on the same page. I understand the `boolean isEmpty()` method needs to be implemented in order to conform to the `Answerable` interface. My concern here is we're doing a `null` check on **value** but, when it's declared, it's annotated with an explicit `@NotNull` contract. If we're performing `null` checks, we're potentially masking bugs which would arise when that contract is being broken, or we're misleadingly stating the property will not contain a `null` value when in reality, it may.

I'm neutral on whether the field should have a `NotNull` contract or not, but we should make a consistent decision one way or the other.

Reply:
Ok, right. I checked the logic that uses this method and set returned value to be always `false`.